appium/tests/specs/live/inbox/CheckMessageProcessingErrors.spec.ts

```diff
@@ -100,5 +100,6 @@ describe('INBOX: ', () => {
     await BaseScreen.checkModalMessage(keyMismatchAttachmentError);
     await EmailScreen.clickDownloadButton();
     await EmailScreen.checkAttachmentTextView(firstAttachmentBody);
+    await EmailScreen.clickBackButton();
```

Comment:
Do we need this back button click here?

Reply:
Yeah. I think this error occurred because we don't navigate back to EmailScreen (and it can't find `key mismatch unexpectedly produces a modal` subject field).

> [iPhone 13 iOS 15.4 #0-14] stale element reference: The previously found element ""key mismatch unexpectedly produces a modal" TextView" is not present in the current view anymore. Make sure the application UI has the expected state. You could also try to switch the binding strategy using the 'boundElementsByIndex' setting for the element lookup. Original error: No matches found for Identity Binding from input {(40:36
>     TextView, {{16.0, 107.0}, {358.0, 43.0}}, label: 'key mismatch unexpectedly produces a modal'40:36
> )}40:36
> [iPhone 13 iOS 15.4 #0-14] stale element reference: The previously found element ""key mismatch unexpectedly produces a modal" TextView" is not present in the current view anymore. Make sure the application UI has the expected state. You could also try to switch the binding strategy using the 'boundElementsByIndex' setting for the element lookup. Original error: No matches found for Identity Binding from input {(40:36
> [iPhone 13 iOS 15.4 #0-14]     TextView, {{16.0, 107.0}, {358.0, 43.0}}, label: 'key mismatch unexpectedly produces a modal'40:36
>

Reply:
But currently click on the back button is the last step, we don't check anything after it.
I think removing it won't affect test passing

<img width="511" alt="Screenshot 2022-05-05 at 14 06 09" src="https://user-images.githubusercontent.com/1062093/166911141-f39f36af-c323-4610-9738-fe9eacf81279.png">

Reply:
Yeah. but above error occurred even before I added that line.
So I added this line to go back to EmailScreen and then error seems to be disappeared.
That lets me think we should navigate back to EmailScreen before we exist testing.
What do you think?

Reply:
Okay, let's leave it for now, but if `CheckMessageProcessingErrors` occasional failures will be back then this line can be removed.